maint_tools/test_docstrings.py

```diff
@@ -14,6 +14,15 @@
     "LogisticRegression.decision_function",
     "Birch.predict",
     "Birch.transform",
+    "KNeighborsClassifier$",
```

Comment:
Are there methods that are not fixed yet ?
If not, you can use `"KNeighborsClassifier"` and  it will cover all methods.

Reply:
All methods have been fixed. I changed with the suggested regex in commit 5192d.